app/packages/files/service-provider-file.ts

```diff
@@ -37,13 +38,21 @@ export default class ServiceProviderFile {
         this.currentUser = currentUser;
         this.fileModel = fileModel;
         this.configuredStorageAddon = configuredStorageAddon;
-        this.userCanDownloadAsZip = configuredStorageAddon.connectedOperationNames
-            .includes(ConnectedOperationNames.DownloadAsZip);
+        this.userCanDownloadAsZip = false;
+        this.canMoveToThisProvider = false;
+        this.getSupportedFeatures();
         this.providerHandlesVersioning = configuredStorageAddon.connectedOperationNames
             .includes(ConnectedOperationNames.HasRevisions);
         this.parallelUploadsLimit = configuredStorageAddon.concurrentUploads;
-        this.canMoveToThisProvider = configuredStorageAddon.connectedOperationNames
-            .includes(ConnectedOperationNames.CopyInto);
+    }
+
+    async getSupportedFeatures() {
+        const externalStorageService = await this.configuredStorageAddon.externalStorageService;
+        // console.log((await externalStorageService).get())
```

Comment:
Probably don't need this comment anymore.

Reply:
Fixed.